lib/bibliothecary/parsers/nuget.rb

```diff
@@ -120,11 +126,38 @@ def self.parse_csproj(file_contents, options: {})
             type: type
           }
         end
+
+        #Followed Target Framework to TFM mapping
+        #according to https://learn.microsoft.com/en-us/dotnet/standard/frameworks#supported-target-frameworks
+        tfm = manifest.locate('Project/PropertyGroup/TargetFramework')&.first&.text || manifest.locate('PropertyGroup/TargetFramework')&.first&.text
+        old_tfm = manifest.locate('Project/PropertyGroup/TargetFrameworkVersion')&.first&.text || manifest.locate('PropertyGroup/TargetFrameworkVersion')&.first&.text
+
+        if tfm
+          target_framework = identify_target_framework(tfm)
+          packages << identify_target_framework(tfm) if target_framework.any?
+        end
+        packages << { name: ".NET", requirement: dotnet_framework_version(old_tfm), type: 'runtime' } if old_tfm
+
         packages.uniq {|package| package[:name] }
       rescue
         []
       end
 
+      def self.parse_sqlproj(file_contents, options: {})
+        manifest = Ox.parse file_contents
+        dsp = manifest.locate('Project/PropertyGroup/DSP')&.first&.text
+        if dsp
+          version = identify_ms_sql_server_version(dsp)
+        else
+          version = nil
+        end 
```

Comment:
```suggestion
    version = identify_ms_sql_server_version(dsp) if dsp
```
the _version_ will be nil if the condition is false.